replacer.go

```diff
@@ -115,12 +115,13 @@ func (r *replacer) ReplaceAll(input, empty string) string {
 			}
 		}
 		if !found {
+			lastWriteCursor = i
 			continue
 		}
 
 		// advance cursor to end of placeholder
-		i = end + 1
-		lastWriteCursor = i
+		i = end
+		lastWriteCursor = i + 1
```

Comment:
Just wondering, which test case does this fix?

Reply:
This fixes placeholders which are directly behind each other:
`replacer_test.go:155: Expected 'val1123öö_äütest-123space value1.2.3.4' got 'val1{asdf}öö_äü{1}space value{mySuper_IP}' for '{test1}{asdf}{äöü}{1}{with space}{mySuper_IP}'`

and it results in each second placeholder to be ignored (if there is no char between the placeholders) because `i` is set to one position too late and this results in one char to be ignored after a matching placeholder.